include/nbl/builtin/hlsl/spirv_intrinsics/arithmetic.hlsl

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_INTRINSICS_ARITHMETIC_INCLUDED_
+#define _NBL_BUILTIN_HLSL_INTRINSICS_ARITHMETIC_INCLUDED_
+
+// For all WaveMultiPrefix* ops, an example can be found here https://github.com/microsoft/DirectXShaderCompiler/blob/4e5440e1ee1f30d1164f90445611328293de08fa/tools/clang/test/HLSLFileCheck/hlsl/intrinsics/wave/prefix/sm_6_5_wave.hlsl
+// However, we prefer to implement them with SPIRV intrinsics to avoid DXC changes in the compiler's emitted code
```

Comment:
need subgroup in the name, again https://github.com/Devsh-Graphics-Programming/Nabla/pull/519/files/1a6e68447b0344e735db02d50fe5cfdf62fcc952..3c19420e94d459d2b35389469a0eaf26a8b65e95#r1293418894